cargo/remote/lock_api-0.3.3.BUILD.bazel

```diff
@@ -0,0 +1,43 @@
+"""
+cargo-raze crate build file.
+
```

Comment:
Whoah this is cool
https://github.com/google/cargo-raze
How much easier does it make generating build stuff?